### PR TITLE
verify_get: handle empty file

### LINKED ISSUE #20: Handle if replit.nix is empty

### PR DESCRIPTION
**Pull request by ryantm**
Why
===
* If a replit.nix file is empty, we don't want to fail, we can just insert what we need to insert.
* closes #20 

What changed
===
* Add unit test for an empty file
* Add template_empty function to get the full empty template we want to insert
* If the root node has no children, splice in the empty template

Test plan
===
* cargo test
* make an empty replit.nix file
* run nix-editor --add pkgs.ncdu
* See it adds the pkg and doesn't error out

### CODE REVIEW

File: src/verify_getter.rs [hunk 1]
```diff
@@ -89,14 +93,23 @@ fn find_or_insert_key_value_with_key(
...
 }
 
+fn template_empty() -> SyntaxNode {
+    let ast = rnix::Root::parse(EMPTY_TEMPLATE);
+    let errors = ast.errors();
+    if errors.len() > 0 {
+        panic!("template_empty had an error: {:#?}", errors)
+    }
+    ast.syntax().first_child().unwrap().clone_for_update()
+}
```
Comment (masad-frost) [inline src/verify_getter.rs:103]: Maybe useless suggestion given the lifetime of a nix editor invocation, but would it make sense to have EMPTY_TEMPLATE_NODE that we just call `clone_for_update()` on?

**Review by masad-frost — APPROVED** [review]
ay ay

**Comment by bradymadden97** [comment]
Thank you!

File: src/verify_getter.rs (hunk 1 above)
Comment (ryantm) [reply src/verify_getter.rs:103]: I think it's okay, because we are only going to ever reach these codepaths once per session: the first time when it is empty or non-exsistent. Also, empty vs non-existent are mutually exclusive so we will really only parse EMPTY_TEMPLATE once.
